### PR TITLE
added : alert in documentation

### PR DESCRIPTION
**Pull request by sanu29**
Alert with different styles added to the documentation

https://user-images.githubusercontent.com/51195476/153724334-c3b06a26-6e01-4890-952b-420c9b85199a.mp4

### CODE REVIEW

**Review by aiman-ansari — APPROVED** [review]
The way you represent your alert component is awesome!!
Good to merge

**Review by Omkar-Ghate — APPROVED** [review]
Great work. Looks good to merge.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (zoef-yt) [inline .vscode/settings.json:3]: this fille can be added in .gitignore.

File: Utils/utilities3.css [hunk 2]
```diff
@@ -48,3 +48,14 @@
 .bg-light       {    background-color: var(--light);        }
 .bg-dark        {    background-color: var(--dark);         }
 
+/* Text colors */
+
+
+.font-color-white       {    color: #fff;              }
```
Comment (zoef-yt) [inline Utils/utilities3.css:54]: Use var here too.

File: documentation/documentation.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    padding-left: 2rem;
+    background-color: var(--secondary-nav-color);
+    font-size: 0.9rem;
+    font-family: 'Inter', sans-serif;
+    position: fixed;
+    left: 0;
+    top: 4rem;
+    height: 100%;
+    overflow-y: scroll;
+    z-index: 999;
```
Comment (zoef-yt) [inline documentation/documentation.css:30]: give z-index like 10 or something.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: documentation/documentation.css (hunk 3 above)
Comment (sanu29) [reply documentation/documentation.css:30]: that is for maximum.. Nothing can come above it

**Pull request merged**